src/Directory.Build.props

```diff
@@ -23,7 +23,8 @@
     <Platforms>AnyCPU;x64</Platforms>
     
     <!-- Defaults for target frameworks and architecture -->
-    <LibraryTargetFrameworks>$(FullFrameworkTFM);netstandard2.0</LibraryTargetFrameworks>
```

Comment:
@crummel is it OK if msbuild does not build full framework on !windows?

Reply:
No, I think that will cause problems - SDK builds on the full framework to support using VS Code with a source-built SDK (issues https://github.com/dotnet/source-build/issues/125 and https://github.com/dotnet/sdk/issues/2249) and has a dependency on MSBuild, so we'll need the full-framework here too.  Would something like an `[En|Dis]ableFullFrameworkBuild` property that defaults to what you need and we can override make sense here?

Reply:
OK, added it back :)

Reply:
Thanks!